### ISSUE TITLE
Cell

### ISSUE THREAD

**Issue by ZengJunHua8203** [unvell/ReoGrid#491]
How do I get columns based on XY coordinates

**Comment by ZengJunHua8203** [comment]
Will reogrid support this method  ? 
GetPosByPoint(Point p)

**Comment by jingwood** [comment]
No, currently there are no public methods that provide this feature. Since the cell's position can change due to scrolling, zooming, and freezing, coordinate translation is a complex process that can only be performed within ReoGrid.
